### PR TITLE
async user agent usinc WKWebView

### PR DESCRIPTION
**Pull request by radko93**
## Description
Async `getUserAgent` method that uses `WKWebView`. Highly inspired by https://github.com/expo/expo/blob/master/packages/expo-constants/ios/EXConstants/EXConstants.m. Expo is actually better because they save the value and don't create WebView again. 

#756 

## Checklist

* [x] I have tested this on a device/simulator for each compatible OS
* [ ] I added the documentation in `README.md`
* [ ] I mentioned this change in `CHANGELOG.md`
* [ ] I updated the typings files (`deviceinfo.d.ts`, `deviceinfo.js.flow`)
* [ ] I updated the dummy web/test polyfill (`default/index.js`)
* [ ] I added a sample use of the API (`example/App.js`)

### CODE REVIEW

**Comment by mikehardy** [comment]
Thanks! I really appreciate it

File: ios/RNDeviceInfo/RNDeviceInfo.m [hunk 1]
```diff
@@ -398,6 +400,32 @@ - (NSDictionary *)powerState
     resolve(@(locationServicesEnabled));
 }
 
+RCT_EXPORT_METHOD(getUserAgent:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
+{
+#if TARGET_OS_TV
+    reject(@"not available");
```
[the remark this answers is not in the document]
Comment (radko93) [reply ios/RNDeviceInfo/RNDeviceInfo.m]: If you go here you can see that https://developer.apple.com/documentation/webkit is not supported by tvOS

**Comment by mikehardy** [comment]
fantastic, v3 coming up as soon as I finish a merge-fest and some reshaping :-)

**Comment by Vision010** [comment]
@radko93 hello, I want to use your changes, but I wonder if this can circumvent iOS's ITMS-90809. I don't upgrade v3, because of my app does not support androidX(emm~ i'm not a RN developer)

**Comment by mikehardy** [comment]
We're on version 5 now. AndroidX is unavoidable and your app can support it easily using the https://github.com/mikehardy/jetifier (which is built in to the react-native CLI even, so it pretty much just works)

Upgrade to modern versions

**Comment by Vision010** [comment]
@mikehardy thanks u, but my App version is ... you know, BOSS disagree redevelopment. I have tried many ways to support android. fuck it!

**Pull request merged**